tendrl/node_agent/provisioner/ceph/plugins/ceph_installer.py

```diff
@@ -252,3 +253,26 @@ def task_status(self, task_id):
             return res_data
         else:
             return None
+
+    def setup(self, provisioner_node_fqdn):
+        url = 'http://%s:%s/setup/' % (
+            provisioner_node, self._CEPH_INSTALLER_API_PORT)
+        resp = self.http.request(
+            self._MGET,
+            url)
+        if resp.status == 200:
+            resp_data = resp.data.decode("utf-8")
+            try:
+                f = open("/tmp/setup.sh", "w")
```

Comment:
Also not sure if `/tmp/setup.sh` is the path we should use. I feel we should have a tendrl specific directory where we create these tendrl specific temporary files and delete them after using it.